spark-commons/src/test/scala/za/co/absa/spark/commons/errorhandling/implementations/submits/ErrorMessageSubmitWithoutColumnTest.scala

```diff
@@ -61,8 +62,8 @@ class ErrorMessageSubmitWithoutColumnTest extends AnyFunSuite {
 
     expectedErrType assertTo result.errType
     expectedErrCode assertTo result.errCode
-    expectedErrMsg assertTo result.errMsg
+    expectedErrMsg assertTo result.errMessage
     expectedErrValuesCol assertTo result.errColsAndValues
-    expectedAdditionalInfo assertTo result.additionInfo
+    expectedAdditionalInfo assertTo result.additionalInfo
   }
 }
```

Comment:
on line 53, the `columnValue` is not used

Reply:
still there

Reply:
Missed it first time...